tests/web3/firefly_getCoverageData0.expected.json

```diff
@@ -25,27 +25,39 @@
         "program": [
           {
             "programCounter": 0,
-            "hitCount": 1
+            "hitCount": 1,
+            "opcode": "PUSH(3)",
+            "argument": "0xffffff"
```

Comment:
I think it would make more sense to have this be `"argument": ["0xff", "0xff", "0xff"]`

Reply:
NVM, this is `PUSH` which takes its arguments inline.